integration/import/esm.mjs

```diff
@@ -0,0 +1,16 @@
+import { strict as assert } from 'assert'
+import { readFileSync } from 'fs'
+import { join, dirname } from 'path'
+import { fileURLToPath } from 'url'
+import * as rx from 'rxjs'
+import * as operators from 'rxjs/operators'
```

Comment:
This is highlighting the new feature that is added in this PR -- currently these lines of code are not possible in 7.0.0-beta.10